cmd/terramate/cli/cli.go

```diff
@@ -186,6 +187,15 @@ func newCLI(args []string, stdin io.Reader, stdout, stderr io.Writer) *cli {
 		return &cli{exit: true}
 	}
 
+	// When we run terramate --version the kong parser just fails
```

Comment:
o.o

Reply:
yup o.O. To double check you can do this:

```
terramate --log-level="info"
2022-03-16T13:13:14+01:00 FTL failed to parse cli args: [--log-level=info] error="expected one of \"version\",  \"run\",  \"plan\",  \"stacks\",  \"generate\",  ..." action=newCli()
```

The error is "failed to parse cli args": https://github.com/mineiros-io/terramate/blob/2a152446ca7d7e0f38ef70c92798e95c713966d4/cmd/terramate/cli/cli.go#L192

Kong is just great :-)...I have no idea how we can make something that has no subcommands work, like just having flags...I tried checking the docs but it is way too big/annoying and I wasn't able to find anything so I just brute forced x_x.